src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -449,6 +449,66 @@ public async Task ProxyTunnelRequest_PortSpecified_NotStrippedOffInUri()
             Assert.True(connectionAccepted);
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP HTTP stack doesn't support .Proxy property")]
```

Comment:
This test method is testing two different things (i.e. both with and without User-Agent header).  It should be split into 2 separate test methods.  Or better, it should use a Theory and arguments sufficient to reduce the complexity/duplicate nature of the test.

Reply:
ok

Reply:
added Theory and simplify test.